src/Uno.UI.Runtime.Skia.Wpf/Input/FrameIdProvider.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Threading;
+
+namespace Uno.UI.Runtime.Skia.Wpf.Input
+{
+	internal static class FrameIdProvider
+	{
+		private static int _currentFrameId;
+
+		internal static uint GetNextFrameId() =>
+			(uint)Interlocked.Increment(ref _currentFrameId);
+	}
```

Comment:
FYI this `FrameIdProvider` is a hack. It's acceptable for now, but the `FrameId` should not only be a sequential number, but should be the id of the frame rendered when the input event was raised by the system. As on skia we do managed the rendering, we should sync them properly.